util/execdetails/execdetails.go

```diff
@@ -141,6 +161,46 @@ func (e *RuntimeStatsColl) Get(planID string) *RuntimeStats {
 	return runtimeStats
 }
 
+// RecordOneCopTask records a specific cop tasks's execution detail.
+func (e *RuntimeStatsColl) RecordOneCopTask(planID, address string,
```

Comment:
Why you do not just use tipb.ExecutorExecutionSummary as the parameter?

Reply:
Good suggestion. Its parameters have been updated now.